GuillotineBinPack.cpp

```diff
@@ -25,15 +25,16 @@ binHeight(0)
 {
 }
 
-GuillotineBinPack::GuillotineBinPack(int width, int height)
+GuillotineBinPack::GuillotineBinPack(int width, int height, bool allowFlip)
 {
-	Init(width, height);
+	Init(width, height, allowFlip);
 }
 
-void GuillotineBinPack::Init(int width, int height)
+void GuillotineBinPack::Init(int width, int height, bool allowFlip)
 {
 	binWidth = width;
 	binHeight = height;
+    binAllowFlip = allowFlip;
```

Comment:
I think the original files used tabs as indentation, though this new code uses spaces, causing a mismatch in indentation above.